### ISSUE TITLE
feature request: eMail-User can change password 

### ISSUE THREAD

**Issue by nudeldudel** [docker-mailserver/docker-mailserver#3363]
### Context

Maybe I just don't found it, but was there never a question, how eMail-User could change their password seperatly? Is there any third-party-solution? Or some common trick? 

OK, every solution should have the target, zu use the setup-script into the container. But I don't found anything about this.

User-Passwords that the admin do not know, I think, this is a essential security-feature, isn't it?

### Description

As I give a Password to a User, that I choos as the Admin, the User might change it.

### Alternatives

Maybe a litte Script, PHP or something? Maybe in al little container...?

### Applicable Users

Admin, but much mor endusers.

### Are you going to implement it?

Yes, because I know the probability of someone else doing it is low and I can learn from it.

### What are you going to contribute?

I could try to create a litte PHP-Script....

**Comment by casperklein** [comment]
tl;dr: no there is currently no way for users to change their passwords.

There were already requests in the past. See [here ](https://github.com/docker-mailserver/docker-mailserver/issues/3092) for example.

There is also [docker-mailserver-admin](https://github.com/docker-mailserver/docker-mailserver-admin), but there wasn't any activity for a while.

**Comment by georglauterbach** [comment]
This project itself does, as @casperklein said, not provide such a functionality. Development at `docker-mailserver-admin` has stalled though. I will close this issue - you may open a new issue at `docker-mailserver-admin` and ask what the status is there.
